charts/kubernetes-agent/templates/nfs-deployment.yaml

```diff
@@ -26,6 +26,20 @@ spec:
       containers:
         - name: {{ printf "%s-nfs" (include "kubernetes-agent.name" .) }}
           image: "octopusdeploy/nfs-server:latest"
+          lifecycle:
+            preStop:
+              exec:
+                command:
+                  - /bin/bash
+                  - '-c'
+                  - >-
+                    echo "Prestop: Delaying shutdown until NFS connections are
+                    closed" > /proc/1/fd/1;  counter=0;  nfsConnection=$(netstat
```

Comment:
redirecting the output shows these message in the Pod logs - is this bad?

Reply:
On the contrary, that's good isn't it?

Reply:
I'd say that's good 👍